src/Asset/EntrypointLookup.php

```diff
@@ -26,9 +30,27 @@ class EntrypointLookup implements EntrypointLookupInterface
 
     private $returnedFiles = [];
 
-    public function __construct(string $entrypointJsonPath)
+    private $configCacheFactory;
+
+    private $debug;
+
+    private $cacheDir;
+
+    public function __construct(string $entrypointJsonPath, array $options = [])
     {
         $this->entrypointJsonPath = $entrypointJsonPath;
+
+        if (isset($options['debug'])) {
+            $this->debug = $options['debug'];
+        } else {
+            $this->debug = true;
+        }
+
+        if (!isset($options['cache_dir'])) {
+            throw new RuntimeException('Cache directory not set.');
```

Comment:
If it's not set, then we basically just need to disable caching. This is needed, at the very least, for backwards compatibility - we can't add a required option/argument to this class without a BC layer. However, we *can* trigger a deprecation warning so that we can make this option required in the future.